helpers/google/api.js

```diff
@@ -181,6 +181,15 @@ var serviceMap = {
             BridgeArnIdentifier: '', BridgeIdTemplate: '', BridgeResourceType: 'b',
             BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'gcp-Storage',
             BridgeCollectionService: 'gcp-buckets', DataIdentifier: 'data',
+        },
+    'AI & ML':
+        {
+            enabled: true, isSingleSource: true, InvAsset: 'models', InvService: 'vertexAI',
+            InvResourceCategory: 'ai&ml', InvResourceType: 'VertexAI models', BridgeServiceName: 'vertexAI',
+            BridgePluginCategoryName: 'AI & ML', BridgeProvider: 'Google', BridgeCall: 'listModels',
+            BridgeArnIdentifier: '', BridgeIdTemplate: '', BridgeResourceType: 'models',
+            BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'AI & ML',
+            BridgeCollectionService: 'vertexai', DataIdentifier: 'data',
```

Comment:
```suggestion
            enabled: true, isSingleSource: true, InvAsset: 'models', InvService: 'vertexAI',
            InvResourceCategory: 'ai&ml', InvResourceType: 'VertexAI models', BridgeServiceName: 'vertexAI',
            BridgePluginCategoryName: 'AI & ML', BridgeProvider: 'Google', BridgeCall: 'listModels',
            BridgeArnIdentifier: '', BridgeIdTemplate: '{name}', BridgeResourceType: 'models',
            BridgeResourceNameIdentifier: 'name', BridgeExecutionService: 'AI & ML',
            BridgeCollectionService: 'gcp-vertexai', DataIdentifier: 'data',
```
we also need to add this in supported collection service